### PR TITLE
Track ImOnline metrics for each validator

### PR DESCRIPTION
**Pull request by mir-nawaz**
## Description
Track ImOnline metrics for each validator in the active set and add to the list item. 

## Have proper tags been added (for bug, enhancement, breaking change)?
- [x] yes

## Does this PR affect any server routes?
- [ ] yes, and they are tested: [enter the % coverage here]
- [ ] yes, and they are not tested: [enter the % coverage here]
- [ ] yes, but I did not run tests
- [x] no

### CODE REVIEW

**Comment by drewstone** [comment]
Awesome will review this soon! Good question for @jnaviask is whether we should be grabbing the last header here? Will have more comments soon!

**Comment by jnaviask** [comment]
Yeah I don't see the user for the header -- we never seem to reference the lastHeader variable anyway. We fetch blocks for the chain globally as part of the `app` variable: `app.chain.block` with `.height` being the latest number.

**Comment by mir-nawaz** [comment]
> Yeah I don't see the user for the header -- we never seem to reference the lastHeader variable anyway. We fetch blocks for the chain globally as part of the `app` variable: `app.chain.block` with `.height` being the latest number.

I have created 3 PR's, which are nested from stacking-ui. So **lastHeader** is used in [last block number](https://github.com/hicommonwealth/commonwealth/pull/326)

**Comment by jnaviask** [comment]
> I have created 3 PR's, which are nested from stacking-ui. So **lastHeader** is used in [last block number](https://github.com/hicommonwealth/commonwealth/pull/326)

Right, we already fetch that via `app.chain.block.height` (stored as `this.app` in most controllers).

**Comment by jnaviask** [comment]
Oh, I see -- you use it specifically to see in terms of who authored it. That makes sense.

**Comment by drewstone** [comment]
@jnaviask yea its for identifying the last block that a validator created.

**Comment by mir-nawaz** [comment]
This PR is for **imOnline** API integration for validators.

File: client/scripts/views/pages/validators/substrate/validator_row.ts [hunk 1]
```diff
@@ -121,22 +127,24 @@ const ValidatorRow = makeDynamicComponent<IValidatorAttrs, IValidatorState>({
...
+        && m('td.val-points', vnode.attrs.eraPoints || ' ')
+      ),
       (!vnode.attrs.waiting
-        && m('td.val-points', vnode.attrs.eraPoints || '-')
+        && m('td.val-last-hash', byAuthor[vnode.attrs.stash] || ' ')
       ),
+      m('td.val-to-be-elected', vnode.attrs.toBeElected
+        ? m(Icon, { name: Icons.ARROW_RIGHT_CIRCLE, size: 'sm' })
+        : ''),
       (vnode.attrs.waiting
```
Comment (drewstone) [inline client/scripts/views/pages/validators/substrate/validator_row.ts:140]: It looks like you replaced the `toBeElected` checks for the `ImOnline` stat, but you didn't carry over the waiting check (instead its being applied to ImOnline which it shouldn't)

Comment (mir-nawaz) [reply client/scripts/views/pages/validators/substrate/validator_row.ts:140]: I have updated `toBeElected ` to Icon, both exist.

**Pull request closed without merging**